app/src/main/java/org/eyeseetea/malariacare/domain/entity/SurveyAnsweredRatio.java

```diff
@@ -141,4 +154,12 @@ public boolean isCompulsoryCompleted(){
         return compulsoryAnswered>=totalCompulsory;
     }
 
+    public static SurveyAnsweredRatio getModelToEntity(SurveyAnsweredRatioDB surveyAnsweredRatioDB){
```

Comment:
domain entity should not convert from model to entity, this a repository responsibility